java/src/main/java/com/saucelabs/saucebindings/examples/VisualTest.java

```diff
@@ -0,0 +1,47 @@
+package com.saucelabs.saucebindings.examples;
+
+import com.saucelabs.saucebindings.SaucePlatform;
+import com.saucelabs.saucebindings.SauceSession;
+import com.saucelabs.saucebindings.options.SauceOptions;
+import com.saucelabs.saucebindings.options.VisualOptions;
+import org.junit.Test;
+import org.openqa.selenium.remote.RemoteWebDriver;
+
+import java.util.Map;
+
+public class VisualTest {
+
+    @Test
+    public void startSession() {
+        // 1. Create Visual Options instance
+        VisualOptions visualOptions = new VisualOptions();
+        visualOptions.setProjectName("My Project")
```

Comment:
I think it's not very user friendly to force a user to `setProjectname()` with a run time exception instead of just forcing it through Intellisense. Have we considered:
`VisualOptions visualOptions = new VisualOptions("My Project");`